src/tensorflow_utils.cc

```diff
@@ -317,4 +318,18 @@ ParseParameter(
   return nullptr;
 }
 
+TRITONSERVER_Error*
+ParseParameter(
+    triton::common::TritonJson::Value& params, const std::string& mkey,
+    std::string* value)
+{
+  std::string value_str;
```

Comment:
Do we need value_str and associated logic?
We can direcly pass `value` variable to GetParameterValue function as it is already string.